lib/post/bloc/post_bloc.dart

```diff
@@ -264,6 +264,12 @@ class PostBloc extends Bloc<PostEvent, PostState> {
             if (!state.hasReachedCommentEnd && state.commentCount == state.postView!.postView.counts.comments) {
               emit(state.copyWith(status: state.status, hasReachedCommentEnd: true));
             }
+            if (event.commentParentId != null) {
+              // If we come here, we've determined that we've already loaded all of the comments.
+              // But we're currently being asked to load some children.
+              // Therefore we will treat this as an error.
+              throw Exception('Unable to load more replies.');
```

Comment:
> Shouldn't this message come from `AppLocalizations`?

This is kind of a philosophical question about exceptions. Most of the time (especially for system/API exceptions) you'd want to map the type/code into a localized string at the application level. This is kind of a gray area since it's an exception we're generating for the sole purpose of showing the user, so should we embed the translated string directly here, or map it elsewhere? I didn't want to answer that for this fix. 😆

> I kind of want to do a pass on the whole code base and create all missing localization strings

That would be awesome! I've kind of been migrating them as I touch parts of the code that are still not localized yet.

Reply:
I didn't look deep into it but I assumed that exception got fed into the toast which is why I brought it up. But yeah I've never had to deal with localization stuff before so I have no idea what the norm is.

Reply:
Yes, in this case the exception does get fed straight into the toast. But what if the exception came from the Lemmy API? We wouldn't have control over the messaging and it wouldn't be localized. I think in general, exception messages are intended for developers, not to be shown directly to the user. Instead you'd set up a mapping to show a more friendly message. That's why this is kind of a unique case because we know the exception is intended directly for the user, and is being thrown in the front-end where we have access to the current culture and localized strings.

Reply:
> I kind of want to do a pass on the whole code base and create all missing localization strings

@ajsosa BTW, if you end up doing this, remember that we have a new global context (`GlobalContext.context`) for retrieving localized strings even when we're not in a standard widget build context (e.g., settings definitions). I think this was keeping us from doing more before.